### DISCUSSION TITLE
How to read object "MultistateInput" using bacnet.read() ?

### DISCUSSION THREAD

**Discussion by saraembeddian** [ChristianTremblay/BAC0#373] [Q&A]
Hello Everyone, i'm just a Noob here, whether this library support to read object "multistateInput" ? i've tried to read it but it show error as unknown object type... 
![Screenshot 2022-12-30 162335](https://user-images.githubusercontent.com/121667675/210062543-5a0d85ef-3059-4a48-a5c4-8204dd1fed71.jpg)
also it would be great if you give an example to write a value on object "multistateOutput" ?
![Screenshot 2022-12-30 162245](https://user-images.githubusercontent.com/121667675/210062557-5f868bb3-8291-4a1c-b6d4-c9c53df046ac.jpg)

Thanks a lot...

**Comment by saraembeddian** [comment]
This is sorted out... seems its "multiStateOutput" not "multistateOutput". Thanks

**Reply by rprronad** [reply]
while True:
        print("ReadProperty-multitSateInput")
        value = bacnet.read("192.168.0.104:47809 multiStateOutput 0 presentValue")
        print(json.dumps(value, indent=2))
        time.sleep(5) 

while True:
        print("ReadProperty-multitSateInput")
        value = bacnet.read("192.168.0.104:47809 multiStateInput 0 presentValue")
        print(json.dumps(value, indent=2))
        time.sleep(5)

These above examples will works fine for both multi state input as well as multi state output,  Just try once
